src/repositories/location-search.repository.ts

```diff
@@ -3,7 +3,7 @@ import { LocationRepositorySearchResponseRootObject } from '../responses';
 
 export class LocationSearch extends Repository {
   public async index(latitude: number, longitude: number, searchQuery?: string): Promise<LocationRepositorySearchResponseRootObject> {
-    const queryOrTimestamp = searchQuery === null ? { timestamp: Date.now() } : { search_query: searchQuery };
+    const queryOrTimestamp = searchQuery === undefined ? { timestamp: Date.now() } : { search_query: searchQuery };
```

Comment:
It's common convention to use `typeof a === 'undefined`.
In this current case it's not important at all, i'm not asking you to refactor this, just share my experience with you.
https://stackoverflow.com/a/4726198/6023909